resources/Trips.py

```diff
@@ -42,22 +42,28 @@ class Route(Resource):
     def get(self):
         return "hi", 200
 
-    def build_json(self, arr):
+    def build_json(self, args):
+
+        # parse info
+        arr = json.loads(args['trip'])
+        # curr = datetime.datetime.utcnow().isoformat() + 'Z'
+        mode = args['mode']
+        tolls = args['tolls']
+
         # initialize json body
         json_data = {
             'origin': { 'location': {'latLng': {}} },
             'destination': { 'location': {'latLng': {}} },
-            "travelMode": "DRIVE",
-            "routingPreference": "TRAFFIC_AWARE",
-            "departureTime": "2022-10-15T15:01:23.045123456Z", #TODO: change to current time
+            "travelMode": mode,
+            "routingPreference": "TRAFFIC_UNAWARE",
+            # "departureTime": curr, 
             "computeAlternativeRoutes": False,
             "routeModifiers": {
-                "avoidTolls": False,
+                "avoidTolls": tolls,
                 "avoidHighways": False,
                 "avoidFerries": False
```

Comment:
Lol you probably want to avoid ferries. Idk

Reply:
eh i was thinking about removing that entire section dont think it matters that much